packages/makeup-modal/index.js

```diff
@@ -4,25 +4,41 @@ var keyboardTrap = require('makeup-keyboard-trap');
 
 var screenreaderTrap = require('makeup-screenreader-trap');
 
+var makeupHoist = require('makeup-hoist-element');
+
 var modalEl;
+var options;
 
 function unmodal() {
   if (modalEl) {
     screenreaderTrap.untrap(modalEl);
-    keyboardTrap.untrap(modalEl); // let observers know the keyboard is now trapped
+    keyboardTrap.untrap(modalEl);
+
+    if (options.hoist) {
```

Comment:
Let's use this pattern for options: https://github.com/makeup/makeup-js/blob/master/packages/makeup-screenreader-trap/src/index.js#L104

Might not be the latest and greatest method, but at least we keep it consistent.

Reply:
Okay I'll use the `Object.assign` syntax